packages/core/test/stateTree.test.ts

```diff
@@ -137,6 +148,7 @@ describe('State tree', function () {
             .map(() => {
                 return new ZkIdentity()
             })
+        let stateTree = new IncrementalMerkleTree(config.stateTreeDepth)
```

Comment:
`const` unless the variable is being re-assigned.

Reply:
[5f4ff50](https://github.com/Unirep/Unirep/pull/243/commits/5f4ff505d0f5fc341c1f75c9e8cdcab1d6c82606)